tools.go

```diff
@@ -0,0 +1,23 @@
+// +build tools
```

Comment:
`dependencies.go` or `install.go`?

Reply:
`dependencies` are ambiguous because of the repo with other dependencies. 
`install` is very generic and confusing with installing what?

tools.go is standard across the `go` community https://github.com/golang/go/issues/25922

Reply:
Let me know.

Reply:
Sounds good.